extensions/shared/src/main/java/app/revanced/extension/youtube/patches/components/DescriptionsFilter.java

```diff
@@ -21,14 +22,12 @@ public DescriptionsFilter() {
         // game section, music section and places section now use the same identifier in the latest version.
         final StringFilterGroup attributesSection = new StringFilterGroup(
                 Settings.HIDE_ATTRIBUTES_SECTION,
-                "gaming_section.",
-                "music_section.",
-                "place_section.",
+                // "gaming_section.", "music_section.", "place_section."
```

Comment:
These filters are no longer used in the latest YouTube, but may be used if the user has patched YouTube 19.05.36 (Or `Disable layout updates` is turned on).

Don't remove it yet, just in case.